TypeCobol.Test/Parser/Performance/Performance.cs

```diff
@@ -368,8 +368,21 @@ public void Part3_FullParsing_TC_DeepTypes()
 
 
 
-
-
+        /// <summary>
+        /// Method for parsinga document.
+        /// </summary>
+        /// <param name="fullPath"></param>
+        /// <param name="options"></param>
+        /// <param name="format"></param>
+        /// <param name="copiesFolder"></param>
+        /// <returns></returns>
+        protected virtual TypeCobol.Parser parseDocument(string fullPath, TypeCobolOptions options, TypeCobol.Compiler.DocumentFormat format, string[] copiesFolder)
```

Comment:
`ParseDocument` to match PascalCase convention.

About the pattern used, you can improve it and avoid these methods marked as `new` and calling base code. You need 4 classes :
- Performance : abstract base class, contains all the tests and an abstract ParseDocument method to initialize the parser
- 3 concrete classes, 1 per mode
  - NoCfg
  - Cfg
  - Dfa

Each concrete class would implement the ParseDocument method to match its own needs and you can override the Antlr test in Cfg and Dfa to make it empty.

Reply:
![Capture](https://user-images.githubusercontent.com/51322634/105512939-06172600-5cd2-11eb-81b4-358e67680198.PNG)
As you can see, `TestMethod1`  is defined once but translates into multiple tests 👍

Reply:
Sorry for PascalCase convention it's the inverse in Java, I will do the change. Also I'll try your pattern because it's true if it work like that it's fine.

Reply:
At first time I did what to suggest, the Performance base class was not abstract, it didn't work, so maybe with the abstract keyword it will work

Reply:
This method seems not working if tests classes are in different assemblies.